package.json

```diff
@@ -50,7 +50,8 @@
   },
   "dependencies": {
     "@carbon/styles": "1.53.0",
-    "lit": "^3.0.0"
+    "lit": "^3.0.0",
+    "mermaid": "^11.2.0"
```

Comment:
This needs to be moved to the `packages/chat/package.json` since it's specific to chat and not the other packages